app/helpers/icons.py

```diff
@@ -14,3 +17,9 @@ def get_icon_template_url(base_url='', with_color=True):
         color_part = "-{r},{g},{b}"
     return f"{get_icon_set_template_url(base_url)}/icons/{{icon_name}}" \
            f"@{{icon_scale}}{color_part}.png"
+
+
+def fetch_and_clean_unlisted_sets():
```

Comment:
I would generalize this function. Something like `split_string`, or maybe you can find something better. The function would then receive a string argument that is being split, and then call it with `os.environ.get('UNLISTED_ICON_SETS')`.